### DISCUSSION TITLE
select - combobox (multi-select)

### DISCUSSION THREAD

**Discussion by Hesesses** [livewire/flux#142] [Feature Requests] [answered]
![image](https://github.com/user-attachments/assets/0acc4caf-887e-47df-8e1e-44b3ab310bd7)

Am I missing something or is it possible to select multiple items?

**Comment by patrykszady** [comment]
Multiselect is not available yet.

**Reply by joshhanley** [reply]
@patrykszady is correct, multiselect isn't available yet. Caleb wasn't happy with the implementation so has held it back until he can get it right.

**Comment by joshhanley** [comment]
Thanks for reporting! We have converted this issue to a feature request for now, so we can focus on fixing any bugs first.

**Comment by joshhanley** [comment, marked as answer]
Closing this as multiselect has been available since v1.0.23. https://fluxui.dev/components/select#multiple-select

**Comment by mikebronner** [comment]
@joshhanley I don't think this should be closed, as the OP was talking specifically about comboboxes, which currently do not support multiple selections.

**Comment by AdamWills** [comment]
Came across this issue as well.

In the Select component, I can either have search OR multiple... but not both yet.
